.github/workflows/release_updates.yml

```
@@ -0,0 +1,24 @@
+# Automatically updates "vX" branches based on GitHub releases. To cut a new
+# release, use the GitHub UI where you enter a tag name and release name of
+# "vX.Y.Z". See https://github.com/jupyterhub/action-k3s-helm/releases.
+#
+#
+name: Release updates
+
+on:
+  release:
+    types: [published, edited]
```

Is edited needed? If it's just the release notes then presumably that won't change the SHA?

---

I think so, what if you edit the tag referenced for a release? I'm not by any means confident, but this was from the actions own example.

I guess I'll prefer to be wrong about needing it rather than wrong about not needing it, but I don't know. Any choice seems fine to me.

Convenience button ;D

```suggestion
    types: [published]
```

---

Let's leave it for now then. I couldn't find any docs on what `edited` actually means :confused:

---

I observed that you can update what a GitHub release refers to as a git commit, so I if that is done this will be needed. If we rely on automation triggered by creating GitHub releases, then I think it could end up useful to have it trigger on edit in the long run, and non-problematic if it triggers on a dummy change since its still refers to a specific git reference.

Ah, hmmm... I wonder what happens if you edit an old GitHub release's text though, yeah, hmm... Well... It comes down to the logic of the referenced github action I guess if its robust.

---

I think the right call is to have edited remain, because they do a comparison check and do nothing if the release isn't more recent in a semver perspective than the current vX reference points to I think.

https://github.com/Actions-R-Us/actions-tagger/blob/c46540b9f3302c3b4e6a7bd622cb478fc98c52be/src/action.ts#L52-L67